setup/security/authentication/oauth/azure/index.md

```diff
@@ -19,13 +21,50 @@ This page instructs you on how to obtain an OAuth 2.0 client ID and client secre
 5. Click "Settings" -> "Keys". Under "Passwords", add a Key Description (eg Spinnaker), set the expiry and then click "Save".
    "Value" will now be populated. This is your client-secret, copy it to a safe place.
 
-## Applying the settings in hal
+## Configure Halyard
 
-Set up oauth2 with azure:
+You may configure Halyard either with the CLI or by manually editing the hal config.
 
-`hal config security authn oauth2 edit --provider azure --client-id (client id from above)  --client-secret (client secret from above)`
+### Hal config
 
-The Tenant ID of your organization is required for Azure OAuth2.0 login. To obtain it:
+```yaml
+security:
+  authn:
+    oauth2:
+      enabled: true
+      client:
+        clientId: # client ID from above
+        clientSecret: client-secret # client secret from above
```

Comment:
This is super minor, but the first thing I did when I read "client ID from above" is ctrl+f for "client ID" on this page, and initially couldn't find it because in step 4 it says "this is the client-id to pass to hal." It won't let me comment on that line since it's not one you edited, but maybe we can change client-id to client ID so it matches what's here (same with client-secret --> client secret?).

Reply:
Good catch! Fixed!